config/routes.rb

```diff
@@ -1,7 +1,21 @@
 Myflix::Application.routes.draw do
   root to: 'pages#front'
   get 'home', to: 'videos#index'
+
+  resources :users, only: [:create, :show]
+  get 'register', to: 'users#new'
+
+  resources :sessions, only: [:create, :destroy]
+  get 'sign_in', to: 'sessions#new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.